app/services/moderator/manage_activity_and_roles.rb

```diff
@@ -78,6 +78,14 @@ def handle_user_status(role, note)
         check_super_admin
         remove_negative_roles
         user.add_role :super_admin
+      when "Tech Admin"
```

Comment:
When we add a tech admin role,  we push along a :single_resource_admin with DataUpdateScript role too. The rationale for this can be found [here](https://github.com/forem/internalEngineering/issues/276#issuecomment-771658292)